src/core/remote.js

```diff
@@ -1121,6 +1121,7 @@ Remote.prototype.requestTransactionEntry = function(options, callback) {
  * @return {Request} request
  */
 
+Remote.prototype.requestTx =
```

Comment:
Should update the jsdoc comments too, so they don't show positional arguments. I forgot to update them last time, and opened a subsequent WIP p.r. fixing them up.